packages/contracts-bedrock/test/L1/SharedLockbox.t.sol

```diff
@@ -61,6 +61,19 @@ contract SharedLockboxTest is CommonTest {
         assertEq(address(sharedLockbox).balance, _lockboxBalanceBefore + _amount);
     }
 
+    function test_unlockETH_paused_reverts(address _caller, uint256 _value) public {
+        // Set the paused status to true
+        vm.prank(superchainConfig.guardian());
+        superchainConfig.pause("test");
```

Comment:
could we use mockCall instead of really pausing? Sounds more consistent with our practices

Reply:
Sadly, is less consistent with OP practices :( . They try to avoid mocked calls on tests if possible